linkerd/docs/config.md

```diff
@@ -414,6 +414,8 @@ configuring tracers via a top-level `tracers` section:
 
 * *tracers* --
   * *kind* -- One of the supported tracers, by fully-qualified class name.
+  * *debugTrace* -- Print all traces to the console. Note this overrides the global
+    `-com.twitter.finagle.tracing.debugTrace` flag. (default: false)
```

Comment:
yeah, where debugTrace itself defaults to false. wasn't sure how explicit to be here. i'll update.